packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -25,9 +27,18 @@ export const DeleteFilesControls = ({
     onCancel,
     onStart,
     taskCounts,
-    tableData,
+    tasks,
   } = useDeleteActionView({ onClose: _onClose });
 
+  const [{ history }] = useStore();
+  const { current } = history;
+  const path = current?.prefix;
+  const tableData = getDeleteActionViewTableData({
+    tasks,
+    taskCounts,
+    path: path ?? '',
```

Comment:
Nit: Move the nullish coalescing up to line 35 and we can keep the shorthand here